mastodon-alt.el

```diff
@@ -331,7 +331,7 @@ This should be ran just before an update."
   (remove-overlays (point-min) (point-max) 'mastodon-alt-update t)
   (let* ((overlay (make-overlay (point-min) (+ (point-min) 1) nil t))
          (width (- (window-width) 1))
-         (update (format-time-string " Update %H:%M "))
+         (update (format-time-string " Update %R "))
```

Comment:
Can you remove this change (I would prefer to have a commit dedicated to melpazoid)

Reply:
It is in its own separate commit? (I almost always follow one-change per commit, sometimes too pedantically) Or do you mean another pull request?